packages/devtools_test/lib/src/mocks/fake_vm_service.dart

```diff
@@ -0,0 +1,391 @@
+// @dart=2.9
+
+import 'dart:async';
```

Comment:
is there a reason all these fake classes need to be in their own file? It is standard Dart practice to have more than one class per file. Why not have a single file called fakes.dart that contains all Fake class impelementations? We have other smaller fakes still in mocks.dart that don't really need their own file, so it makes sense to keep all the fakes together in a single file.

Reply:
They are copied from mocks.dart. It make sense for large fakes to stay in own file. I will copy small ones mack to mocks.dart after migration.